scripts/spack/configs/linux_ubuntu_20/spack.yaml

```diff
@@ -116,7 +116,7 @@ spack:
       - spec: cmake@3.20.4
         prefix: /usr
     cppcheck:
-      version: [1.90]
+      version: [1.90']
```

Comment:
```suggestion
      version: [1.90]
```

Reply:
apparently the 1.90 converts to 1.9 and gives an error for external users?  The ' is supposed to make it a string.  I have no real way to test this, so thought I'd put it up to see if anyone had thoughts.

Reply:
weird I've never seen a string there but there are examples in the Spack repo but it requires the whole string to be quoted like such:

```
version: ['1.90']
```